examples/ui/navigation/pubspec.yaml

```
@@ -0,0 +1,19 @@
+name: navigation_samples
+description: Sample navigation code.
+publish_to: none
+
+environment:
+  sdk: ^3.5.0
```

Dart 3.6 is out now and I updated the other example projects, so can update this one to be consistent:

```suggestion
  sdk: ^3.6.0
```

---

Thanks for catching this. I've updated my VS Code settings to always have one line break.